sentences/ro/_common.yaml

```diff
@@ -54,14 +54,15 @@ expansion_rules:
   este: "(e | este)"
   modifica: "(schimb(ă|a) | modific(ă|a) | seteaz(ă|a))"
   in: "((î|i)n)"
-  din: "(din | (î|i)n | pentru | de la)"
+  din: "(din | (î|i)n | pentru | de la | de pe)"
```

Comment:
Check out the latest commits https://github.com/home-assistant/intents/blob/main/sentences/ro/_common.yaml#L57

Trebuie un pull